artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-sa
+  namespace: kube-apiserver-storage
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcdproxy-crd-clusterrole
+rules:
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages/status"]
+  verbs: ["get", "watch", "list", "create", "update", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: etcdproxy-role
+  namespace: kube-apiserver-storage
+rules:
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "watch", "list", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "watch", "list", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["secrets", "configmaps"]
+  verbs: ["get", "watch", "list", "create", "update", "patch", "delete"]
```

Comment:
reduce the verbs

Reply:
> reduce the verbs

To what?

Reply:
@deads2k This is already fixed. This was from the previous commit, before we rebased the PR. For some reason, GitHub has not disbanded the comment after rebase.